resourcecontrollerv2/resource_controller_v2_examples_test.go

```diff
@@ -668,6 +668,7 @@ var _ = Describe(`ResourceControllerV2 Examples Tests`, func() {
 			Expect(resourceInstance).ToNot(BeNil())
 		})
 		It(`DeleteResourceInstance request example`, func() {
+			fmt.Println("\nDeleteResourceInstance() result:")
```

Comment:
We don't actually need this label since we're displaying only the status code below after the "end" comment.